docs/forwarding-events-to-destinations/forwarding-events/google-tag-manager-server-side/snowplow-client-for-gtm-ss/snowplow-client-configuration/index.md

```diff
@@ -4,32 +4,32 @@ date: "2021-11-24"
 sidebar_position: 100
 ---
 
-## Populating common User Data
-
-The [GTM Common Event](https://developers.google.com/tag-platform/tag-manager/server-side/common-event-data) has a `user_data` property. To populate this, you can attach a context Entity to your events of this schema: `iglu:com.google.tag-manager.server-side/user_data/jsonschema/1-0-0`, which can be found on [Iglu Central](https://github.com/snowplow/iglu-central/blob/853357452300b172ebc113d1d75d1997f595142a/schemas/com.google.tag-manager.server-side/user_data/jsonschema/1-0-0).
-
 ## Forward User IP address
 
 As the container sits between the website user and the Snowplow collector (or other downstream destinations), the users IP will be unknown to the destination. By enabling this option, the users IP address will be included in the events sent to Tags.
 
 By disabling this, you are able to use GTM SS as a proxy which can string user IP addresses from requests. Many tags also offer this functionality at the tag level.
 
+## Populate GAv4 Client Properties
+
+Enabled by default, this option will populate additional properties which the GAv4 requires, that is useful if you want to forward your Snowplow events to the GAv4 Tag.
+
+### Populating common User Data
```

Comment:
Why is this a H3 now? It's not related to the above GAv4 Client Properties, unless I'm misremembering.

Reply:
Since it is not a configuration option, i wasn't sure where it belongs. That's why i moved it there, thinking that it is relevant to populating client properties. But you are right, it is not related to where it is now. Do you think there could be a section for such information? Or i can just move it back, because it was definitely clear. What do you think?

Reply:
Oh I see, it was more like advice. Maybe put it in a `:::tip` at the top?

Reply:
Done! Tips look great!